tools/privacyidea-token-janitor

```diff
@@ -106,9 +245,17 @@ def _get_tokenlist(last_auth, assigned, active, tokeninfo_key,
         if description and not re.search(description,
                                          token_obj.token.description):
             continue
-        if tokeninfo_value and tokeninfo_key and not re.search(tokeninfo_value,
-                                                               token_obj.get_tokeninfo(tokeninfo_key)):
-            continue
+        if tokeninfo_value_filter and tokeninfo_key:
+            value = token_obj.get_tokeninfo(tokeninfo_key)
+            # if the tokeninfo value is not even set, it does not match the filter
```

Comment:
I think the comment should be 

   if the tokeninfo *key* is not even set.

Because if the key was set, but had no value, it would return an empty string, not None.